docs/en/latest/plugins/multi-auth.md

```diff
@@ -41,6 +41,7 @@ For Route:
 | Name         | Type  | Required | Default | Description                                                           |
 |--------------|-------|----------|---------|-----------------------------------------------------------------------|
 | auth_plugins | array | True     | -       | Add supporting auth plugins configuration. expects at least 2 plugins |
+|hide_credentials|	boolean|	False|	false|	Set to true will not pass the authorization request of header\query\cookie to the Upstream.|
```

Comment:
the "`|`" looks really misplaced and you could improve the formatting a little (without changing other lines ofcourse)